### DISCUSSION TITLE
SetGesture이 작동이 안됩니다.

### DISCUSSION THREAD

**Discussion by bestdeveloperjo** [naverz/zepeto-studio-kor#1176] [5. Q&A]
해당 코드에서 반복된 애니메이션이 나오게 loop로 설정을 해놓았지만 애니메이션이 시작조차 안됩니다.

export default class NPC extends ZepetoScriptBehaviour {

        private zepetocharacter : ZepetoCharacter;
        private isSpawn : bool = false;
        public Tmp : GameObject;
        public ZepetoName : string;
        public ani : AnimationClip;
        Start() {
            let spawn : SpawnInfo = new SpawnInfo()
            spawn.position = this.transform.position;
            spawn.rotation = this.transform.rotation;
            ZepetoCharacterCreator.CreateByZepetoId(this.ZepetoName, spawn, (character : ZepetoCharacter)=>{
                this.zepetocharacter = character
                this.isSpawn = true;
                this.zepetocharacter.SetGesture(this.ani);
               
                this.zepetocharacter.tag = "Player"
            });
            
            //console.log(LangTrans.GetText(1));
            
        }

   
    
    
    
    }

**Comment by zlmrzlo** [comment]
혹시 인스펙터 상에서 애니메이션클립이 들어가 있는 상태인가요? 그것을 먼저 확인해봐야 할 것 같네요.
그게 아니라면 this.zepetocharacter.SetGesture(this.ani); 부분을 start 함수 마지막에 넣어서 실행해보는 것은 어떨까요?

**Reply by bestdeveloperjo** [reply]
애니메이션은 들어가 있습니다. 
start 함수 마지막에 넣어보니 
TypeError: 정의되지 않은 'SetGesture' 속성을 읽을 수 없습니다. 라는 내용이 뜹니다.

**Reply by zlmrzlo** [reply]
정의되지 않았다는 것은 임포트를 적절한 것으로 하지 않아서 생긴 문제 같은데 혹시 임포트 제대로 되었는지도 알 수 있을까요?

**Comment by YSHFM** [comment]
https://www.mettit.com/pt/view/PRE20220527165937414
해당 글에서 애니메이션 동기화 및 애니메이션 재생부분을 다루고 있어서 참고해보시면 도움이 되실 것 같습니다~
